pkg/interfaces/contracts/standalone-utils/IBalancerFeeBurner.sol

```diff
@@ -14,6 +14,7 @@ interface IBalancerFeeBurner is IProtocolFeeBurner {
     struct SwapPathStep {
         address pool;
         IERC20 tokenOut;
+        bool isBuffer;
     }
 
```

Comment:
```suggestion
```
This is now identical to the BatchRouter definition, so we can remove this and just import it:

`import { SwapPathStep } from "../vault/BatchRouterTypes.sol";`

Reply:
+1; if it's exactly the same let's reuse. It's also more clear for a reader if you're already familiar with the router.